qiskit/circuit/classical/expr/constructors.py

```diff
@@ -181,11 +188,16 @@ def logic_not(operand: typing.Any, /) -> Expr:
             >>> expr.logic_not(ClassicalRegister(3, "c"))
             Unary(\
 Unary.Op.LOGIC_NOT, \
-Cast(Var(ClassicalRegister(3, 'c'), Uint(3)), Bool(), implicit=True), \
+Cast(Var(ClassicalRegister(3, 'c'), Uint(3)), \
+Bool(), implicit=True), \
```

Comment:
super minor: are you able to stop whatever tool it is you have from modifying the line-wrapping when it's not necessary?  Just adds a bit more noise in review

Reply:
It's actually me, and I won't typically do this 🤣. It's an artifact of moving const-ness from `Type` to `Expr` and having to manually update the comments (they needed wrapping when the types included their const-ness in their initializers).